fetch/api/response/response-consume.html

```diff
@@ -12,17 +12,31 @@
   </head>
   <body>
     <script>
+    function responseStringToMultipartFormTextData(response, name, value) {
+        assert_true(response.headers.has("Content-Type"), "Response contains Content-Type header");
+        var boundaryMatches = response.headers.get("Content-Type").match(/;\s*boundary=("?)([^";\s]*)\1/);
+        assert_true(!!boundaryMatches, "Response contains boundary parameter");
+        return stringToMultipartFormTextData(boundaryMatches[2], name, value);
+    }
+
+    function stringToMultipartFormTextData(multipartBoundary, name, value) {
+      return ('--' + multipartBoundary + '\r\n' +
+              'Content-Disposition: form-data; name="' + name + '"\r\n' +
```

Comment:
I guess this is ok for now.
I wonder whether we will not want something like:
Content-Disposition: form-data;name...